go.mod

```diff
@@ -7,10 +7,11 @@ module github.com/hyperledger/aries-framework-go
 require (
 	github.com/agl/ed25519 v0.0.0-20170116200512-5312a6153412
 	github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d
-	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/golang/protobuf v1.3.1 // indirect
```

Comment:
Why we need golang/protobuf

Reply:
Just reran 'go mod tidy' and it disappeared. Not sure why it was there in the first place, but it's gone now.